metrotranscom/doorway
# feat: add command to reseed with external listings

# Pull Request Template

## Issue Overview


## Description

This PR adds two new reseed options, `yarn db:reseed:with-external:dev` and `yarn db:reseed:with-external:prod`, and a couple of supporting actions in the `package.json` file for `backend/core`.  The regular `yarn db:reseed` will now emit a notice if called directly, informing the user that they probably want to use one of the two new options instead.

## How Can This Be Tested/Reviewed?

Run `yarn db:reseed:print-notice` to view the notice that gets displayed when run.
Run `yarn db:reseed` to verify that the notice displays when run.
Run `yarn db:reseed:with-external:prod` to verify that listings are imported from the prod Bloom endpoint and that no notice is displayed.
Run `yarn db:reseed:with-external:dev` to verify that listings are imported from the dev Bloom endpoint and that no notice is displayed.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
